### PR TITLE
Userdocs revamping

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
## Description of the Pull Request (PR):

Replaces much of references to Singularity and updates. 

## This fixes or addresses the following GitHub issues:

- Fixes #

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
-Copyright (c) 2018-2020, Sylabs, Inc. All rights reserved.
+Copyright (c) 2018-2020, Apptainer. All rights reserved.
```
Comment (dtrudg) [inline LICENSE:2]: Sylabs produced most of the content of these documents. Apptainer project copyright should be added for the appropriate time frame. Sylabs copyright should not be removed.

Note below that a condition of the BSD license (which Sylabs added on this repo) is that:

> Redistributions of source code must *retain the above copyright notice*, this
list of conditions and the following disclaimer.

Comment (pedroalvesbatista) [reply LICENSE:2]: Oh sure, some of the replacements was done automatically, but this part maybe I left behind, sorry. Will fix it.

Comment (DrDaveD) [reply LICENSE:2]: Agreed

Comment (dtrudg) [reply LICENSE:2]: Cheers :+1:

**Comment by DrDaveD** [comment]
Closing, as updates are now instead being made on https://github.com/apptainer/apptainer-userdocs

**Pull request closed without merging**